cluster-autoscaler/simulator/drain.go

```diff
@@ -29,18 +29,17 @@ import (
 	schedulerframework "k8s.io/kubernetes/pkg/scheduler/framework/v1alpha1"
 )
 
-// FastGetPodsToMove returns a list of pods that should be moved elsewhere if the node
+// FastGetPodsToMove returns a list of pods that should be moved elsewhere and a list of DaemonSet pods that should be evicted if the node
```

Comment:
Nit: the line length in the func comments seems to extend a bit long. See if you can wrap it.